content/pt-br/platform-as-a-service.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Plataforma como serviço (PaaS)
+status: Completed
+category: Tecnologia
+tags: ["fundamentos", "plataforma", ""]
+---
+
+## O que é
+
+Plataforma como serviço, ou PaaS, é uma plataforma externa para equipes de desenvolvimento de aplicativos implantarem e executarem seus aplicativos. Heroku, Cloud Foundry, App Engine são exemplos de ofertas de PaaS.
+
+## Problema relacionado
+
+Para aproveitar os padrões nativos da nuvem, como [microsserviços](https://glossary.cncf.io/microservices/) ou [aplicativos distribuídos](https://glossary.cncf.io/distributed-apps/), as equipes de operações e os desenvolvedores precisam ser capazes de descarregar uma grande quantidade de operações e trabalho de manutenção. Isso inclui tarefas como provisionamento de infraestrutura, manipulação de [descoberta de serviço](https://glossary.cncf.io/service-discovery/), balanceamento de carga e [escalabilidade](https://glossary.cncf.io/pt-br/scalability/) de aplicativos.
```

Comment:
For the links you can use just:
`[microsserviços](/microservices/)` instead of the full URL.

Reply:
Done on all links